test/integration/testpaths/testpath_usage.py

```diff
@@ -2879,6 +2882,60 @@ def test_07_positive_tests_usage(self):
         # aggregation period and current period will give the network usage
         return
 
+    @attr(tags=["advanced", "basic"], required_hardware="false")
+    def test_08_checkNewVolumein_listUsageRecords(self):
+        """ Test case to check if new volume crated after
+        restore VM is listed in listUsageRecords
+        # 1. Launch a VM
+        # 2. Restore the VM
+        # 3. Check if the new volume created is listed in listUsageRecords API
+        """
+
+        # Step 1
+        vm_cluster = VirtualMachine.create(
+            self.userapiclient,
+            self.testdata["small"],
+            templateid=self.template.id,
+            accountid=self.account.name,
+            domainid=self.account.domainid,
+            serviceofferingid=self.service_offering.id,
+            zoneid=self.zone.id,
+        )
+
+        volumes_root_list = list_volumes(
+            self.apiclient,
+            virtualmachineid=vm_cluster.id,
+            type='ROOT',
+            listall=True
+        )
+
+        root_volume = volumes_root_list[0]
+
+        # Step 2
+        vm_cluster.restore(self.apiclient)
+
+        qresultset = self.dbclient.execute(
+            "select id from volumes where name='%s' and state='Ready';" %
+            root_volume.name)
+
+        self.assertNotEqual(
+            len(qresultset),
+            0,
+            "Check DB Query result set"
+        )
+
+        volumeCheck = "Volume Id: " + str(qresultset[0][0]) + " usage time"
+
+        response = self.listUsageRecords(usagetype=6)
+        self.assertEqual(response[0], PASS, response[1])
+        UsageRecords = [record for record in response[1]
+                        if volumeCheck in record.description]
+        # Step 3
+        self.assertNotEqual(
```

Comment:
even  if  UsageRecords is empty test case will pass , is that we are expecting ?